### ISSUE TITLE
x/build: UI improvements for subrepo benchmarks

### ISSUE THREAD

**Issue by findleyr** [golang/go#59337]
This issue tracks UI improvements following up on #53538. That issue reached a natural stopping point, as the benchmarks at perf.golang.org/dashboard are useful for the tools team but there is more lower priority UI work that could be done.

Specifically: 

- [ ] Links to commits are be broken (they go to the Go repo). Fix those.
- [ ] We probably also want some UI distinction that these are x/tools benchmarks and thus the x axis and baseline is completely unrelated to other benchmarks.
- [ ] Because runs will trigger on changes to the Go repo, we will end up with duplicate points at the same experiment commit date (overlapping on the x axis). We should visually deduplicate these somehow (perhaps only render the first one we find).

CC @prattmic

**Comment by prattmic** [comment]
Commit links are already correct, those were fixed a while ago.

I'd also add that links around the UI almost all drop the existing URL query parameters, which is quite painful for the tools results, as most links are broken, trying to go to some tools benchmark but in the go repo.

**Comment by findleyr** [comment]
> Commit links are already correct, those were fixed a while ago.

Right, thanks! Added your suggestion (and yes, that is a pain -- I've learned to only visit the shortlink I created).

Also: feel free to edit the issue to add whatever you want.
